airflow/providers/facebook/ads/hooks/ads.py

```diff
@@ -105,24 +110,79 @@ def bulk_facebook_report(
         params: Dict[str, Any],
         fields: List[str],
         sleep_time: int = 5,
-    ) -> List[AdsInsights]:
+    ) -> Union[List[AdsInsights], Dict[str, List[AdsInsights]]]:
         """
-        Pulls data from the Facebook Ads API
+        Pulls data from the Facebook Ads API regarding Account ID with matching return type
+
+        .. seealso::
+            If the type of Account ID is a List account_id -> list
```

Comment:
This `->` syntax is a bit difficult to understand to me. It’s probably better to write this is prose, something like

```rst
The return type and value depends on the ``account_id`` configuration. If the
configuration is a str representing a single Account ID, the return value is the
list of reports for that ID. If the configuration is a list of str representing
multiple Account IDs, the return value is a dict of Account IDs and their
respective list of reports.
```

Reply:
Also this part does not belong in ``seealso``, which is generally used to reference external content.

Reply:
I am grateful to you for your time to review too many changes and your help. I have changed the comment same as your comment since you have already described it very well.